web/themes/custom/cloudy/source/_patterns/01-components/links/_icon-link.twig

```diff
@@ -1,8 +1,18 @@
 {% set wrapper_classes = wrapper_classes|default([]) %}
 {% set class = 'class=' ~ wrapper_classes|join(' ') %}
+{% set inner_classes = inner_classes|default([
+  'flex',
+  'items-center',
+  'mb-4',
+]) %}
+{% set icon_classes = icon_classes|default([
+  'flex',
+  'items-center',
+  'mr-2',
+])%}
 <a href="{{ url }}" {{ wrapper_class_markup ? class }}>
-  <div class="flex items-center mb-4">
```

Comment:
This code is going to conflict with POWR-1311. When a document is embedded, we want as few DIVs around the content as possible so that it can be placed inline within a paragraph or in a list (ordered or unordered). @brittanysmart may have something to add based on her story.

Reply:
We could make the wrapper element configurable, but this still won't place the document link inline within paragraph text. See [POWR-1311](https://portland.atlassian.net/browse/POWR-1311) for details.